src/Navbar.tsx

```diff
@@ -313,7 +313,7 @@ class Navbar extends React.Component<any, State> {
                   return (
                     <NavLink
                       key={topLevelTo}
-                      className={`top-nav-button nav-button nav-button-container center-content ${this.getBorder(
+                      className={`top-nav-button nav-button-container center-content ${this.getBorder(
```

Comment:
I thought we might need the nav-button class for desktop view, does the site look okay without it?

Reply:
Yes, the top-nav-button has the desktop css also.